### PR TITLE
Changes make aha work

### PR DESCRIPTION
**Pull request by joyliu37**
I make the following modification to enable h2h work with clockwork memory mapping in aha. 
One change that I made is changing CLOCKWORK_PATH and COREIR_DIR to be absolute path. Because verilator did not support relative path. And the `hardware_target.mk` assume the path the relative path.  Let me know if this work for your original test. 
I also comment out the verilator test and sanity check.

### CODE REVIEW

File: apps/hardware_benchmarks/hw_support/Makefile.inc [hunk 1]
```diff
@@ -13,7 +13,7 @@ include $(HALIDE_BIN_PATH)/apps/support/Makefile.inc
 # For CoreIR generation
 COREIR_CXX_FLAGS = -I $(COREIR_DIR)/include -L$(COREIR_DIR)/lib -Wl,-rpath,$(COREIR_DIR)/lib
 COREIR_CXX_FLAGS += -I $(FUNCBUF_DIR)/include -L$(FUNCBUF_DIR)/bin -Wl,-rpath,$(FUNCBUF_DIR)/bin
-COREIR_CXX_FLAGS += -I $(CLOCKWORK_DIR) -L$(CLOCKWORK_DIR)/lib -Wl,-rpath,$(CLOCKWORK_DIR)/lib  
+COREIR_CXX_FLAGS += -I $(CLOCKWORK_DIR) -L ${CLOCKWORK_DIR} -L$(CLOCKWORK_DIR)/lib -Wl,-rpath,$(CLOCKWORK_DIR)/lib
```
Comment (jeffsetter) [inline apps/hardware_benchmarks/hw_support/Makefile.inc]: Can you change the {brackets} to (parentheses) for consistency?

File: apps/hardware_benchmarks/hw_support/hardware_targets.mk [hunk 2]
```diff
@@ -149,7 +149,7 @@ $(BIN)/optimized_$(TESTNAME).cpp opt-clockwork clockwork-opt opt: $(BIN)/clockwo
 	cd $(BIN) && LD_LIBRARY_PATH=../$(CLOCKWORK_PATH)/lib:../$(COREIR_DIR)/lib ./clockwork_codegen opt >/dev/null; cd ..
 compile_mem compile-mem mem-clockwork clockwork-mem mem: $(BIN)/clockwork_codegen
 	@-mkdir -p $(BIN)/coreir_compute && cp $(BIN)/$(TESTNAME)_compute.json $(BIN)/coreir_compute/$(TESTNAME)_compute.json
-	cd $(BIN) && CLKWRK_PATH=../$(CLOCKWORK_PATH) LD_LIBRARY_PATH=../$(CLOCKWORK_PATH)/lib:../$(COREIR_DIR)/lib ./clockwork_codegen compile_mem >mem_cout; cd ..
+	cd $(BIN) && CLKWRK_PATH=$(CLOCKWORK_PATH) LD_LIBRARY_PATH=$(CLOCKWORK_PATH)/lib:$(COREIR_DIR)/lib ./clockwork_codegen compile_mem ; cd ..
```
Comment (jeffsetter) [inline apps/hardware_benchmarks/hw_support/hardware_targets.mk]: Can you check that this works without $CLOCKWORK_PATH defined? I usually don't have $CLOCKWORK_PATH or $COREIR_DIR defined, which leads them to use relative paths.

Comment (jeffsetter) [reply apps/hardware_benchmarks/hw_support/hardware_targets.mk]: Also, please leave the outputting> into a file.

**Review by jeffsetter — CHANGES_REQUESTED** [review]

**Comment by joyliu37** [comment]
Merge your changes. Can you double check if it's working.

**Review by jeffsetter — APPROVED** [review]

**Pull request merged**